PocketKit/Sources/Sync/PocketSource.swift

```diff
@@ -35,9 +35,28 @@ public class PocketSource: Source {
     private let notificationObserver = UUID()
 
     private let operations: SyncOperationFactory
-    private let syncQ: OperationQueue = {
+    private let saveQueue: OperationQueue = {
         let q = OperationQueue()
+        // need to save data to the server 1 at a time cause a user can favorite, then unfavorite in a specific order.
         q.maxConcurrentOperationCount = 1
+        q.qualityOfService = .background
+        q.name = "com.mozilla.pocket.save"
+        return q
+    }()
+
+    private let fetchSavesQueue: OperationQueue = {
+        let q = OperationQueue()
+        q.maxConcurrentOperationCount = 1
+        q.qualityOfService = .background
+        q.name = "com.mozilla.pocket.fetch.saves"
+        return q
+    }()
+
+    private let fetchArchiveQueue: OperationQueue = {
+        let q = OperationQueue()
+        q.maxConcurrentOperationCount = 1
+        q.qualityOfService = .background
+        q.name = "com.mozilla.pocket.fetch.archive"
         return q
```

Comment:
Medium: These could be consolidated into one function that takes the name as a parameter.